drivers/quota/projectquota.go

```diff
@@ -166,7 +168,7 @@ func (q *Control) SetQuota(targetPath string, quota Quota) error {
 	//
 	// set the quota limit for the container's project id
 	//
-	logrus.Debugf("SetQuota(%s, %d): projectID=%d", targetPath, quota.Size, projectID)
+	logrus.Debugf("SetQuota(%s, %s): projectID=%d", targetPath, quota, projectID)
```

Comment:
This should probably also log the inode limit.

Reply:
Is should show the entire struct now, Not just the size.